### PR TITLE
presence indicators

### PR DESCRIPTION
**Pull request by akash1810**
The positioning isn't consistent across all pages, but I think there's benefit in shipping this now.

Also refactors out the publish state element into its own component to position the indicators in the right place on the video page.

# Search page
Nothing, obviously.
image

# Video page
image

# Upload page
image

# Audit page
image

# Help page
image

### CODE REVIEW

File: app/controllers/VideoUIApp.scala [hunk 1]
```diff
@@ -77,16 +77,12 @@ class VideoUIApp(val authActions: HMACAuthActions, conf: Configuration, awsConfi
   }
 
   private def presenceConfig(user: User): Option[Presence] = {
-    (awsConfig.stage, conf.getString("presence.domain")) match {
-      case ("DEV", _ ) =>
```
Comment (akash1810) [inline app/controllers/VideoUIApp.scala]: allow presence in def if config exists

File: public/video-ui/styles/components/_presence.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.presence-list {
```
Comment (akash1810) [inline public/video-ui/styles/components/_presence.scss]: stolen from atom workshop...

File: public/video-ui/src/components/Presence.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 
 export class Presence extends React.Component {
-  state = null;
+  state = {
+    client: null,
+    visitors: []
```
Comment (akash1810) [inline public/video-ui/src/components/Presence.js]: this felt like the simplest thing to do, rather than throw it to the store... because I'm lazy 😄

Comment (shaundillon) [reply public/video-ui/src/components/Presence.js]: It's easy enough to push up if it's needed elsewhere. Local state is for this kind of thing anyway

**Pull request merged**